gulpfile.js

```diff
@@ -0,0 +1,54 @@
+const { task, src, series, dest } = require('gulp');
+const log = require('fancy-log');
+const modifyFile = require('gulp-modify-file');
+const prettier = require('gulp-prettier');
+
+task('check-wiki.json', done => {
+  src('./wiki.json')
+    .pipe(
+      modifyFile(content => {
+        let hasWrongFormatName = false;
+        const wikiData = JSON.parse(content);
+
+        for (let idx = 0; idx < wikiData.length; idx++) {
+          if (isValidName(wikiData[idx].name) === false) {
+            hasWrongFormatName = true;
+            break;
+          }
+        }
+
+        const sorted = !!hasWrongFormatName
+          ? wikiData
+          : wikiData.sort((a, b) => {
+              return a.name < b.name ? -1 : a.name > b.name ? 1 : 0;
+            });
```

Comment:
```suggestion
          : wikiData.sort((a, b) => a.localeCompare(b));
```

[String.prototype.localeCompare](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) 써봐도 좋을 것 같아요!

Reply:
@greatSumini 이런게 있군요!